op-node/node/api.go

```diff
@@ -68,6 +69,20 @@ func (n *adminAPI) SequencerActive(ctx context.Context) (bool, error) {
 	return n.dr.SequencerActive(ctx)
 }
 
+// PostUnsafePayload is a special API that allows posting an unsafe payload to the L2 derivation pipeline.
+// It should only be used by op-conductor for sequencer failover scenarios.
+func (n *adminAPI) PostUnsafePayload(ctx context.Context, payload *eth.ExecutionPayload) error {
+	recordDur := n.M.RecordRPCServerRequest("admin_postUnsafePayload")
+	defer recordDur()
+
+	if actual, ok := payload.CheckBlockHash(); !ok {
+		log.Error("payload has bad block hash", "bad_hash", payload.BlockHash.String(), "actual", actual.String())
+		return fmt.Errorf("payload has bad block hash: %s, actual block hash is: %s", payload.BlockHash.String(), actual.String())
+	}
+
+	return n.dr.OnUnsafeL2Payload(ctx, payload)
+}
```

Comment:
This feels like something we should have a test for.  We could write an e2e test that starts two nodes with no gossip connection, stop the batcher (so safe head doesn't change) and should be able to use this API to progress the unsafe head of the non-sequencer.

Reply:
Agree having tests would be a good practice. 

However, from the current test setup, no other methods from admin API has its own tests. I think it make sense because they're just super thin wrappers to functions of `rollup.Driver`, and that is extensively tested (Same as our `OnUnsafeL2Payload` function)

Reply:
They do have e2e tests like https://github.com/ethereum-optimism/optimism/blob/49d82402d521c0288021b3ffee2b35e1d002a4f2/op-e2e/system_adminrpc_test.go#L17

Reply:
Thanks, added tests per suggestion. Weird that perviously my code intel didn't find me any references of calls to admin API